ATSQuadroStrategyBase/NinjaTrader 8/bin/Custom/Strategies/AlgoSystemBase.cs

```diff
@@ -2511,7 +2511,7 @@ public virtual StrategyTradeWorkFlowState ProcessWorkFlow(StrategyTradeWorkFlowS
                     }
                     break;
                 case StrategyTradeWorkFlowState.GoLongPlaceProfitTargetsPending:
-                    if (IsHistoricalTradeOrPlayBack || !IsSubmitTargetsAndConfirm)

```

Comment:
This is required -  IsHistoricalTradeOrPlayBack  it is taking the fastest route there is not confirm process in replay